src/callbacks_step/save_restart.jl

```diff
@@ -133,6 +133,18 @@ function load_time(restart_file::AbstractString)
 end
 
 
+"""
+    load_timestep(restart_file::AbstractString)
+
+Load the time step saved in a `restart_file`.
+"""
```

Comment:
```suggestion
"""
    load_timestep(restart_file::AbstractString)

Load the time step number (`iter` in OrdinaryDiffEq.jl) saved in a `restart_file`.
"""
```
I think the original formulation may have been a bit ambiguous since it could also mean the time step size `dt`. Feel free to choose another formulation to clarify this.